src/utils/config.js

```diff
@@ -0,0 +1,11 @@
+// Configs, used in renderer process
+'use strict'
+const Config = require('electron-config')
```

Comment:
`electron-config` is not installed in https://github.com/egoist/eme/blob/dev/package.json , it's installed as `./app`'s [dependency](https://github.com/egoist/eme/blob/dev/app/package.json),  while in fact we can use pass the `config` object from the main process to the render process, just like how we set the `$state` [here](https://github.com/egoist/eme/blob/dev/app/package.json)